mlops/orchestration/dbt/analytics/models/trulens/trulens_simple.sql

```diff
@@ -0,0 +1,34 @@
+-- A simplified table for displaying trulens data
+
+WITH raw_data AS (
+    SELECT 
+        record_id,
+        app_id,  
+        input,
+        output,
+        ts,
+        record_json::json AS record_json
+    FROM 
+        {{ source('public', 'records') }}
+    WHERE
+        app_id = 'TopK_ReRank_v1'
```

Comment:
you may want to capture all the app id I would think.

Reply:
Hmm, good point. Why are we filtering by the app_id to begin with in these view creations? Should we just remove that entirely? The other .sql file has it as well.